src/executor.js

```diff
@@ -17,12 +20,26 @@ const executor = async () => {
     return;
   }
 
+  try {
```

Comment:
Is there a reason to `try/catch` here and not for the entire `executor`?

Reply:
Sure, we could wrap the entire executor in a `try/catch`if that makes more sense.

Reply:
What do you think? I was really just trying to get your thought process.

Reply:
I think it makes more sense to have the `try/catch` here isolated to just the `getCookieCutterSource`. Wrapping the entire executor in a `try/catch` would make it difficult to know where an error actually comes from.